src/TypeScriptGenerator.ts

```diff
@@ -3,7 +3,7 @@ import {
   TypeGenerator,
   TypeGeneratorOptions
```

Comment:
I think we can just import IRVisitor, SchemaUtils here and get rid of RelayCompilerPublic.